bin/o2a-run-sys-test-complete

```diff
@@ -17,7 +17,7 @@ _SHORT_OPTIONS="
 h a: p: C: L: b: c: m: r: z: d: v A S W K
 "
 
-_ALLOWED_PHASES=" prepare-configuration convert prepare-dataproc test-composer test-oozie "
+_ALLOWED_PHASES=" prepare-configuration convert prepare-dataproc test-composer test-oozie compare-artifacts "
```

Comment:
Update docs

Reply:
Done. Thanks.

Reply:
May I ask why you are reviewing you own PRs and thanking yourself? 🤔

Reply:
Before bedtime, I did a review. I wanted to keep my thoughts, so I left comments. In the morning I solved my comments and did not want to close the discussions without comment.